bindings/cron/cron_test.go

```diff
@@ -64,61 +69,19 @@ func TestCronInitFailure(t *testing.T) {
 
 // TestLongRead
 // go test -v -count=1 -timeout 15s -run TestLongRead ./bindings/cron/.
-func TestCronReadWithDeleteInvoke(t *testing.T) {
```

Comment:
Aren't `TestCronInitSuccess` and `TestCronInitWithSeconds` the same? It's unrelated to this PR, but maybe we can follow table driven tests for check for different formats within a single `TestCronInit` method,

Reply:
removed these tests and replaced with a common test containing various cron format and schedules